mappings/net/minecraft/data/server/BlockLootTableGenerator.mapping

```diff
@@ -1,67 +1,104 @@
+		ARG 0 drop
+	METHOD method_10377 oreDrops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1792;)Lnet/minecraft/class_52$class_53;
+		ARG 0 dropWithSilkTouch
+		ARG 1 drop
+	METHOD method_10378 oakLeavesDrop (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;[F)Lnet/minecraft/class_52$class_53;
+		ARG 0 leaves
+		ARG 1 drop
+		ARG 2 chance
+	METHOD method_10380 dropsWithShears (Lnet/minecraft/class_2248;Lnet/minecraft/class_79$class_80;)Lnet/minecraft/class_52$class_53;
+		ARG 0 drop
 		ARG 1 child
-	METHOD method_10381 create (Lnet/minecraft/class_2248;Lnet/minecraft/class_4570$class_210;Lnet/minecraft/class_79$class_80;)Lnet/minecraft/class_52$class_53;
+	METHOD method_10381 drops (Lnet/minecraft/class_2248;Lnet/minecraft/class_4570$class_210;Lnet/minecraft/class_79$class_80;)Lnet/minecraft/class_52$class_53;
+		ARG 0 drop
 		ARG 1 conditionBuilder
 		ARG 2 child
-	METHOD method_10382 createForBlockWithItemDrops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;)Lnet/minecraft/class_52$class_53;
-		ARG 1 lootWithoutSilkTouch
-	METHOD method_10383 createForSlabs (Lnet/minecraft/class_2248;)Lnet/minecraft/class_52$class_53;
-	METHOD method_10384 create (Lnet/minecraft/class_1935;Lnet/minecraft/class_59;)Lnet/minecraft/class_52$class_53;
+	METHOD method_10382 drops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;)Lnet/minecraft/class_52$class_53;
+		ARG 0 dropWithSilkTouch
+		ARG 1 drop
+	METHOD method_10383 slabDrops (Lnet/minecraft/class_2248;)Lnet/minecraft/class_52$class_53;
+		ARG 0 drop
+	METHOD method_10384 drops (Lnet/minecraft/class_1935;Lnet/minecraft/class_59;)Lnet/minecraft/class_52$class_53;
+		ARG 0 drop
 		ARG 1 count
-	METHOD method_10385 createForLargeMushroomBlock (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;)Lnet/minecraft/class_52$class_53;
-		ARG 1 loot
-	METHOD method_10386 createForBlockWithItemDrops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;Lnet/minecraft/class_59;)Lnet/minecraft/class_52$class_53;
-		ARG 1 lootWithoutSilkTouch
+	METHOD method_10385 mushroomBlockDrops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;)Lnet/minecraft/class_52$class_53;
+		ARG 0 dropWithSilkTouch
+		ARG 1 drop
+	METHOD method_10386 drops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1935;Lnet/minecraft/class_59;)Lnet/minecraft/class_52$class_53;
+		ARG 0 dropWithSilkTouch
+		ARG 1 drop
 		ARG 2 count
-	METHOD method_10387 createForCropStem (Lnet/minecraft/class_2248;Lnet/minecraft/class_1792;)Lnet/minecraft/class_52$class_53;
-		ARG 1 seeds
-	METHOD method_10388 createForNeedingSilkTouchShears (Lnet/minecraft/class_2248;Lnet/minecraft/class_79$class_80;)Lnet/minecraft/class_52$class_53;
+	METHOD method_10387 cropStemDrops (Lnet/minecraft/class_2248;Lnet/minecraft/class_1792;)Lnet/minecraft/class_52$class_53;
+		ARG 0 stem
+		ARG 1 drop
+	METHOD method_10388 dropsWithSilkTouchShears (Lnet/minecraft/class_2248;Lnet/minecraft/class_79$class_80;)Lnet/minecraft/class_52$class_53;
```

Comment:
Just curious, what block uses this?

Reply:
Cobwebs and the nether vines....which makes me realize there's a slight error here. it should be silk touch OR shears. Fixed in latest commit.